app/serializers/api/admin/reports/order_serializer.rb

```diff
@@ -0,0 +1,63 @@
+class Api::Admin::Reports::OrderSerializer < ActiveModel::Serializer
```

Comment:
@sauloperez the basic idea is that the user loads a report as html, and then a separate request for report data is sent via AJAX to `ReportsController` which uses the `lib/open_food_network` classes to load data for displaying in different classes of report, this data is then serialised and rendered as JSON, which is consumed by the ui-grids library and rendered dynamically on the page.